tools_docker/test_docker/README.md

```diff
@@ -0,0 +1,45 @@
+# Docker Tests
+
+This is a tool to help test Docker images, for simple cases, but also for complicated tests.
+This allows:
+- automatically running Tracim with Collabora and Elasticsearch on by default (it just needs some tweaks
+for emails and you need to setup dns resolving using a special Docker image)
+- testing if the Docker container correctly starts
+- automatically testing services running in the docker container
+
+## Prerequisities
+
+- install some python packages:
+`pip install pytest testinfra python-dotenv`
+- build tracim docker image `DOCKER_TRACIM_IMAGE` (see test_config.env)
+- build elasticsearch docker image `DOCKER_ELASTICSEARCH_IMAGE`
+- create tracim env config file from sample:`cp tracim.sample.env tracim.env`
+- create docker test config file from sample: `cp test_config.sample.env test_config.env`
+
+for hostname resolving from host, use:
+
+```sh
+docker run --name dns_docker_solver --hostname dns.mageddo --restart=unless-stopped -p 5380:5380 \
+-v /var/run/docker.sock:/var/run/docker.sock \
+-v /etc/resolv.conf:/etc/resolv.conf \
+defreitas/dns-proxy-server
+```
+## Automatic testing
+
+Tests can now be started:
+
+```sh
+pytest test_docker
+```
+
+It will check in the Docker container if all the Tracim services are correctly running after launching the Tracim, Collabora and ElasticSearch Docker containers
+
```

Comment:
Missing information about what is going on when:
- test ok
- test not ok

Error visible directly or in log file ? if I start test with `docker run -d`, where I can seen if its working or not ?

Reply:
These are standard python test. We do have a python test (like backend test) that run multiple docker and return error in console according to test, i don't adapt more the docker runned but we may want to improve log for manual test error (where error are in tracim docker log file).